src/resolve/resolved-varlink.c

```diff
@@ -554,11 +597,44 @@ int manager_varlink_init(Manager *m) {
                 return log_error_errno(r, "Failed to attach varlink connection to event loop: %m");
 
         m->varlink_server = TAKE_PTR(s);
+
+        if (m->enable_varlink_notifications) {
+                if (m->varlink_notification_server)
+                        return 0;
+
+                r = varlink_server_new(&s, VARLINK_SERVER_ACCOUNT_UID);
+                if (r < 0)
+                        return log_error_errno(r, "Failed to allocate varlink server object: %m");
+
+                varlink_server_set_userdata(s, m);
+
+                r = varlink_server_bind_method_many(
+                                s,
+                                "io.systemd.Resolve.Monitor.SubscribeDnsResolves", vl_method_subscribe_dns_resolves);
```

Comment:
Not a matter, but please consistently break lines...
```c
                r = varlink_server_bind_method_many(
                                s,
                                "io.systemd.Resolve.Monitor.SubscribeDnsResolves",
                                vl_method_subscribe_dns_resolves);
```
or
```c
                r = varlink_server_bind_method_many(
                                s, "io.systemd.Resolve.Monitor.SubscribeDnsResolves", vl_method_subscribe_dns_resolves);
```

Reply:
Fixed